client/state/post-types/taxonomies/test/reducer.js

```diff
@@ -205,6 +206,26 @@ describe( 'reducer', () => {
 			} );
 		} );
 
+		test( 'should accumulate items for multiple sites and post types', () => {
+			const actions = [
+				receivePostTypeTaxonomies( 2916284, 'page', [] ),
+				receivePostTypeTaxonomies( 2916285, 'page', [] ),
```

Comment:
does this line add anything to the test?

Reply:
I test both accumulating multiple sites and multiple post types. There's no existing test for either case. Or did you mean something else?

Reply:
I suppose we could add some test content, so the expected output is a little more obvious? At first glance this looks like an extra copy pasta, but notice that the second call has a different `siteId`

Reply:
I guess I was thinking that there are two tests here:

```js
// would have passed before this pr
test('can accumulate post-types taxonomies for multiple sites')
```

and

```js
// would have failed before this pr
test('can update the post-type taxonomies for a single site')
```